### ISSUE TITLE
(med) Alternative text - What to look for section needs clarification

### ISSUE THREAD

**Issue by brianelton** [w3c/wai-easy-checks#178]
Location: [Alternative Text - What to look for](https://deploy-preview-131--wai-easychecks.netlify.app/test-evaluate/easy-checks/image-alt/#what-to-look-for)

**Current wording:** 

Most points are written as the usual things to look for when evaluating alt text, but the decorative image bullet refers to what you would see when the bookmarklet is run and it's sort of confusing.

What to look for

- Images with information relevant to the page’s content have alternative text that decribes that important information
- Images with text should have the text in the alternative text
- Images that appear to be just to be decorative should be marked with ‘Alt text = “”. Suitable?’
- Images that are functional, like a link or a button, should have alternative text that describes the destination page or the function of the button
- Complex images like graphs and charts have short alternative text that describes the type of image and a short summary of the key point
  - Complex images should have the detail described elsewhere on the page or on a separate page with a link

**Suggested revision:**

What to look for

- A yellow box will be adjacent to each image on a page with the associated alternative text and a prompt to note whether the text is suitable
  - Images with information relevant to the page’s content should have alternative text that describes that important information
  - Images with text should have the text in the alternative text
  - Images that are decorative should be noted as having an empty alt text attribute ('Alt text="". Suitable?')
  - Images that are functional, like a link or a button, should have alternative text that describes the destination page or the function of the button
  - Complex images like graphs and charts should have short alternative text that describes the type of image and a short summary of the key point
    - Complex images should have the detail described elsewhere on the page or on a separate page with a link
 
**Rationale**: for clarity in understanding the bookmarklet functionality and consistency in how it is written. (all bullets should use "should" or none of them should).

Also note that "describes" is misspelled as "decribes" in the first bullet point and should be fixed regardless of the other suggestions.

**Comment by iadawn** [comment]
Agree with the changes to make use of 'should' consistent.

Disagree with including a description of the functionality of the bookmarklet. If we start that here, we would need to include it everywhere. And I am not convinced it is that necessary. (Slightly) open to persuasive evidence.

**Comment by brianelton** [comment]
@iadawn Just want to make sure that the bullet about decorative is captured because it is slightly re-written to align with the others. "Images that are decorative should be noted as having an empty alt text attribute ('Alt text="". Suitable?')"

I can live without the addition describing what the bookmarklet does.
